src/routes/docs/[component]/+layout.svelte

```diff
@@ -19,9 +19,9 @@
 	{@html theme}
 </svelte:head>
 
-<div class="flex grid-cols-12 flex-col gap-8 overflow-hidden py-6 lg:grid lg:px-4">
+<div class="flex grid-cols-12 flex-col gap-8 overflow-hidden py-6 lg:grid lg:px-6">
 	<div class="col-span-2">
-		<ul class="flex w-full overflow-x-auto px-4 pb-2 lg:block lg:px-0">
+		<ul class="flex w-full flex-col gap-2 overflow-x-auto p-2">
```

Comment:
There shouldn't be a gap here, IMO. MAkes it possible to click between links, resulting in a non-action, which is frustrating. I agree it can be visually strange, but I see it in other places too 
![image](https://user-images.githubusercontent.com/26071571/234230855-3f4d0e28-1b1e-4b48-bcec-eed4cc558f30.png)

Reply:
made similar change on this pr https://github.com/TGlide/radix-svelte/pull/109

i do agree it looks a bit weird on mobile for some reason.

<img width="557" alt="Screenshot 2023-04-25 at 10 47 09" src="https://user-images.githubusercontent.com/20280618/234240012-7807b891-b376-434e-b51a-1832ea35eeab.png">

Reply:
Good point, will likely also move this to a menu as well. 

@jca41 I'll implement the changes here to avoid merge conflicts, thanks for pointing that out.

Reply:
I think a separate menu action for mobile devices would work better, will implement that in a separate PR

Reply:
@rjt-rockx Alright, seems good